tests/IntegrationTest.php

```diff
@@ -12,9 +12,9 @@ class IntegrationTest extends PHPStanTestCase
      */
     public function dataIntegrationTests(): iterable
     {
-        yield [__DIR__.'/integration/data/test-case-extension.php'];
-        yield [__DIR__.'/integration/data/model-properties.php'];
-        yield [__DIR__.'/integration/data/blade-view.php'];
+        yield [__DIR__.'/data/test-case-extension.php'];
+        yield [__DIR__.'/data/model-properties.php'];
+        yield [__DIR__.'/data/blade-view.php'];
```

Comment:
Why this change? `integration` folder is good because then it's clear these are data for integration tests.

Reply:
It follows the pattern established in other layers of the test folder hierarchy. There, it is not clear to which test class the data belongs to either:

![image](https://user-images.githubusercontent.com/12158000/221528450-7ef4476e-cc1b-4087-a942-d9f34e3609eb.png)

Reply:
So maybe it's better to also put the `IntegrationTest` into some `Integration` folder. And in there have `data` folder.

Reply:
Just did that, thanks. I also renamed `tests/Application` to `tests/application` to signify it is not just another namespace for tests to live in.